meta-hovergames-distro/recipes-hovergames/images/imx-image-hovergames-demo.bb

```diff
@@ -376,18 +378,26 @@ fakeroot do_update_dns() {
 	set +x
 }
 
-fakeroot do_enable_network_manager() {
+do_fix_connman_conflict() {
 	set -x
 
-	# In bionic, but not in xenial. We want all [network] interfaces to be managed
-	# so that we do not have to mess with interface files individually
-	if [ -e "${APTGET_CHROOT_DIR}/usr/lib/NetworkManager/conf.d/10-globally-managed-devices.conf" ]; then
-		sed -i -E "s/^unmanaged-devices\=\*/unmanaged-devices\=none/g" "${APTGET_CHROOT_DIR}/usr/lib/NetworkManager/conf.d/10-globally-managed-devices.conf"
-	fi
+	#rm ${IMAGE_ROOTFS}/etc/systemd/system/network-online.target.wants/NetworkManager-wait-online.service
+	#rm ${IMAGE_ROOTFS}/etc/systemd/system/dbus-org.freedesktop.nm-dispatcher.service
+	#rm ${IMAGE_ROOTFS}/etc/systemd/system/multi-user.target.wants/NetworkManager.service
+	rm ${IMAGE_ROOTFS}/etc/systemd/system/dbus-org.freedesktop.resolve1.service
+	rm ${IMAGE_ROOTFS}/etc/systemd/system/multi-user.target.wants/systemd-resolved.service
```

Comment:
We do an apt install, right? I wonder if it is possible to apt uninstall?

Reply:
This is a trick to disable the systemd-resolved service. It's the technically the same what the "systemctl disable systemd-resolved" command does. We cant uninstall systemd-resolved because it's part of systemd.

Reply:
Is it not related to the network manager? I thought removing network manager might help.

Reply:
No it's not, first I thought Network-manager caused the problems that's why I initially commented it out. But those commented lines should be removed.